bundle/config/mutator/python/python_locations.go

```diff
@@ -116,6 +117,10 @@ func parsePythonLocations(input io.Reader) (*pythonLocations, error) {
 			return nil, fmt.Errorf("failed to parse python location: %s", err)
 		}
 
+		if !pathlib.IsAbs(entry.File) {
```

Comment:
What are the guarantees for paths coming from Python?

Reply:
Thanks. Is this captured in a comment somewhere in this file?

Reply:
Fixed now